plugins/node/opentelemetry-instrumentation-bunyan/README.md

```diff
@@ -118,6 +118,10 @@ const logger = bunyan.createLogger({
 });
 ```
 
+## Semantic Conventions
+
+This package uses `@opentelemetry/semantic-conventions` version `1.22+`, which implements Semantic Convention [Version 1.7.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.7.0/semantic_conventions/README.md)
+
```

Comment:
I kinda think this should be dropped. The only use a semconv in this package is in the *tests* to use the `SERVICE_NAME` constant.

Reply:
See https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2182#pullrequestreview-2052361574

Reply:
done